Dockerfile

```diff
@@ -38,7 +38,13 @@ RUN cd /opt/cs3 && \
 
 RUN fix-permissions "/home/${NB_USER}"
 
-ENV JUPYTER_ENABLE_LAB = 1
```

Comment:
The idea was not to set the env variables in the image but to start the extension with default configs.
The difference is that the extension should not require these extra configs even if installed in a different docker image (which will be the case)